SpringBox/pair_interactions.py

```diff
@@ -2,57 +2,75 @@
 import numpy as np
 from scipy.spatial.distance import pdist, squareform
 
+
 def active_particles(pXs, prv_acc, activation_fn, _config):
     # Particles that were previously active, now can loose their activation with a rate of activation_decay_rate
     prv_acc = prv_acc * (np.random.rand(len(prv_acc)) > _config['activation_decay_rate'] * _config['dt'])
     # Or particles can become active when they enter the light activated area
     new_acc = activation_fn(pXs)
     return np.maximum(prv_acc, new_acc)
 
+
 @numba.jit
-def spring_forces(acc,pXs , Dij, _config):
+def spring_forces(acc, pXs, Dij, Drxns, _config):
```

Comment:
What does Drxns stand for?
otherwise, I'd suggest dpXs (for difference pXs)

Reply:
I was going for directions but dpXs makes sense too. Possibly Dpij? Because it gives the vectors between points i and j?

Reply:
Sounds good to me